service/src/test/resources/application-integration-test.yml

```diff
@@ -13,7 +13,7 @@ evaka:
     fee_decisions: trevaka-paymentdecisions-it
     voucher_value_decisions: trevaka-vouchervaluedecisions-it
   database:
-    url: jdbc:postgresql://localhost:5432/evaka_it
+    url: jdbc:postgresql://localhost:5432/evaka_tampere_it
```

Comment:
I think we could remove this line now, better to not have default value for url (and rely on `application-<municipality>_evaka.yaml` in tests).

Reply:
Also removed